### PR TITLE
[FEAT] Turtles

### PR DESCRIPTION
**Pull request by Rofrtd**
# Description

This PR implements 2 turtles, Emerald Turtle and Tin Turtle.
turtles

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/game/types/collectibles.ts [hunk 1]
```diff
@@ -104,6 +106,28 @@ export const GOBLIN_PIRATE_ITEMS: Record<
...
+      "Iron Compass": new Decimal(10),
+      Seaweed: new Decimal(50),
+      "Block Buck": new Decimal(1),
+    },
+    sfl: new Decimal(100),
+    supply: 100,
+  },
+  "Tin Turtle": {
+    description:
+      "The Tin Turtle gives +0.3 to to Stones you mine within its Area of Effect.",
```
Comment (brian-man) [inline src/features/game/types/collectibles.ts:122]: typo: "to to"

Comment (Rofrtd) [reply src/features/game/types/collectibles.ts:122]: Thank you!

File: src/features/game/events/landExpansion/moveGold.test.ts [hunk 2]
```diff
@@ -122,4 +122,43 @@ describe("moveGold", () => {
       },
     });
   });
+
+  it("does not move gold if mined and within Turtle AeO", () => {
```
Comment (brian-man) [inline src/features/game/events/landExpansion/moveGold.test.ts:126]: typo: AeO -> AOE

File: src/features/game/events/landExpansion/moveIron.test.ts [hunk 3]
```diff
@@ -122,4 +122,43 @@ describe("moveIron", () => {
       },
     });
   });
+
+  it("does not move iron if mined and within Turtle AeO", () => {
```
Comment (brian-man) [inline src/features/game/events/landExpansion/moveIron.test.ts:126]: same typo

File: src/features/game/events/landExpansion/moveStone.test.ts [hunk 4]
```diff
@@ -122,4 +122,43 @@ describe("moveStone", () => {
       },
     });
   });
+
+  it("does not move gold if mined and within Turtle AeO", () => {
```
Comment (brian-man) [inline src/features/game/events/landExpansion/moveStone.test.ts:126]: same typo

**Review by adamhannigan — DISMISSED** [review]
LGTM

**Pull request merged**